sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
 
-Azure.Core is a reusable library used to implement .NET Azure SDK components. 
-It contains APIs that make it easy to call Azure service endpoints in a convinient, performant, and consistent manner.
+Azure.Core provides shared primitives, abstractions, and helpers for .NET Azure SDK service client libraries. 
+These shared APIs allow client libraries to expose common functionality in a consistent fashion, 
+so that once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
 
-# Installing
+The main shared concepts of Azure.Core (and so all Azure SDK libraries using Azure.Core) include:
```

Comment:
nit:  kindof.... maybe if we're looking into the future.   Today, Azure.Core is very HTTP/REST centric and doesn't map well to many of the scenarios outside of that.   I'm not suggesting to change it, per say, but I'm not saying that softening that guarantee would be a bad idea either.

Reply:
I removed "all"